src/client/utils/overriding.js

```diff
@@ -9,5 +9,11 @@ export default function overrideDescriptor (obj, prop, getter, setter) {
     if (setter !== null)
         descriptor.set = setter;
 
+    return descriptor;
+}
+
+export function overrideDescriptor (obj, prop, getter, setter) {
```

Comment:
1)For more readbility, let's rename `overrideDescriptor` and change its signature to
```js
overrideObjectProperty(obj, prop, descriptor); 
....

Usage:

overrideObjectProperty(window.Response.prototype, 'type', {
     get: function () {}
});
```
2)Let's combine 
`createOverriddenDescriptor` and `DocumentSandbox._definePropertyDescriptor` 
into single util function named as 
`overrideObjectPropertyInNestedProto (obj, nestedObjProto, prop, descriptor)`

Usage:
```js
overrideObjectPropertyInNestedProto (domainPropertyOwner, htmlDocPrototype, 'domain', {
    get: function () {}
});
```